### PR TITLE
Export using workflow versions

### PR DESCRIPTION
**Pull request by marten**
Do not merge until we've backfilled production.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by camallen** [comment]
a first pass on this look ok - specs are failing though i haven't looked into why.

**Comment by marten** [comment]
Ah a test that factories up an invalid classification without a workflow now crashes because the factory can't get the workflow version without a workflow... :P Will fix

File: spec/factories/classifications.rb [hunk 1]
```diff
@@ -11,11 +11,11 @@
                  {task: "another_one",
                   value: [1, 2]}]
     user_ip "192.168.0.1"
-    workflow_version "15.15"
     completed true
     user
     project
     workflow
+    workflow_version { workflow ? "#{workflow.major_version}.#{workflow.minor_version}" : nil }
```
Comment (camallen) [inline spec/factories/classifications.rb]: Does this inherit from the workflow association on the line above? for some reason i assume this had to happen in the after build hooks, nice to know not.

Comment (marten) [reply spec/factories/classifications.rb]: Yep, this works fine and even you pass in a specific workflow (as evidenced by the test failing on `create :classification, workflow: nil`)

**Pull request merged**